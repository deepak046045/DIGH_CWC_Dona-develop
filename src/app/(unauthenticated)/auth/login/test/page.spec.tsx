import { render } from '@testing-library/react';
import React from 'react';
import Login from '../page';
import '@testing-library/jest-dom';

jest.mock('next-auth/react');

describe('Login Page', () => {
  it('should render page', () => {
    const { container } = render(<Login />);
    expect(container).toMatchSnapshot();
  });
});
