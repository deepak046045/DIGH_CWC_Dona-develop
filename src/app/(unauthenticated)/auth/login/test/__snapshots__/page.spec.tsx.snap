// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Page should render page 1`] = `
<div>
  <div>
    <h1
      class="h1"
    >
      Redirecting to login...
    </h1>
    <lds-loading-spinner
      animationspeed="1000"
      arialabel="loading"
      size="40"
    />
  </div>
</div>
`;
